backend/Recodo.API.BLL/Services/RequestService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.Net.Http.Headers;
+using Recodo.FIle.BLL.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.FIle.BLL.Services
+{
+    public class RequestService : IRequestService
+    {
+        private static HttpClient Client { get; set; } = new HttpClient();
+
+        private static string BaseUrl = new string(@"https://localhost:44316/api/");
+
+        public async Task<string> SendSaveRequest(string token)
+        {
+            Client.DefaultRequestHeaders.Add(HeaderNames.Authorization, token);
```

Comment:
static HttpCLient + DefaultRequestHeaders? Can you make HttpClient scoped at least?

Reply:
Scoped this service in startup? Or I should declare HttpClient inside a method?